packages/cli/src/lib/implementation/validate-plugin-filter-options.utils.ts

```diff
@@ -1,8 +1,14 @@
 import type { CategoryConfig, PluginConfig } from '@code-pushup/models';
 import { filterItemRefsBy, ui } from '@code-pushup/utils';
 
+export class OptionValidationError extends Error {
+  constructor(message: string) {
+    super(`${message}`);
+  }
+}
```

Comment:
You can omit the `constructor` if it's 1:1 with extended class:

```suggestion
export class OptionValidationError extends Error {}
```